packages/nest-pg-notify/README.md

```diff
@@ -0,0 +1,15 @@
+# nest-pg-notify
```

Comment:
README is empty. What does this package do? Pretty sure we want this to be published on NPM.

Why do I (as someone looking to use this package) need ioredis and redlock?

Reply:
See my comment on the PR

`Still needs documentation but you can get the gist of it.`

Reply:
Whoops, missed that!